### DISCUSSION TITLE
How to use click events in the latest version

### DISCUSSION THREAD

**Discussion by yangliguo7** [pixijs/pixijs#7978] [Q&A] [answered]
Recently, I am studying Pixi!
I cant capture the click event.
I check the code
[https://pixijs.io/examples/#/events/drag-n-drop.js](url)
`
if (!('events' in app.renderer)) {
    app.renderer.addSystem(PIXIEvents, 'events');
}
`
But how can I get PIXIEvents?

version`"pixi.js": "^6.1.3"`

**Comment by yangliguo7** [comment]
`
import { Application, Sprite,Renderer } from 'pixi.js';

import * as PixiEvent from    "@pixi/events";

if (!('events' in app.renderer)) {
    app.renderer.addSystem(PixiEvent.EventSystem, 'events');
}

app.stage.addEventListener('pointerdown', function handleClick() {
    console.log('Hello world!');
});

`
I cant capture the event.Q_Q~
is that true?

**Comment by yangliguo7** [comment, marked as answer]
I resolved it !

**Reply by bkhebert** [reply]
how?!
